inst/tutorials/r_and_rstudio_basic/r_and_rstudio_basic.Rmd

```diff
@@ -33,31 +33,42 @@ By the end of this tutorial you should be able to:
 - Recognize and use functions.
 - Install and load R packages.
 - Load and subset tabular data using tidyverse.
-- Use the `help` function in R console to troubleshoot given a new function.
-The last bullet point can be more descriptive saying, "Use the `help` function in R console to troubleshoot and *identify mandatory parameters* given a new function."
+- Use the `help` function in R console to troubleshoot and identify required arguments for a given function
 
 
 ## A Tour of RStudio
-
+By the end of this section, you will be able to:
+- Name the three panes in RStudio and what they do
+- Change the sizes of the panes
+- Navigate through the console using common keyboard shortcuts
+- Change the appearance of RStudio
 When you start RStudio, you will see something like the following window appear:
 
 ![](/images/rstudio.png){width=100%}
 
 Notice that the window has three "panes":
 
-- Console (lower left side): this is your view of the R engine. You can type in R commands here and see the output printed by R. (To tell them apart, your input is in blue, and the output is black.) There are several editing conveniences available: use up and down arrow keys to go back to previously entered commands, which you then can edit and re-run; TAB for completing the name before the cursor; see more in [online docs](http://www.rstudio.com/ide/docs/using/keyboard_shortcuts).
+- Console (lower left side): this is your view of the R engine. You can type in R commands here and see the output printed by R. (To tell them apart, your input is in blue, and the output is black.) There are several editing conveniences available: up and down arrow keys to go back to previously entered commands which you then can edit and re-run, TAB for completing the name before the cursor, and so on. See more in [online docs](http://www.rstudio.com/ide/docs/using/keyboard_shortcuts).
```

Comment:
I would maybe suggest that the Console is for quick exploration, and is handy in, say, printing a data frame or checking the length of a vector or something, if you haven't addressed this later.